src/components/Contact.js

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from "react";
+import EmailValidator from "email-validator";
+//import { useForm, Controller } from "react-hook-form";
+
+function Contact() {
+  let [text, setText] = useState("");
+
+  const validateEmail = (e) => {
+    console.log("Current email is: ", e.target.value);
+    const emailValidator = EmailValidator.validate(e.target.value);
+    if (emailValidator === true) {
+      console.log("Email is valid.");
+      setText("");
+    } else {
+      setText("Email not valid. Please enter a valid email.");
+      console.log("Info: ", text);
+    }
+  };
+
+  const validateName = (e) => {
+    const fullName = e.target.value.trim();
+    console.log("Full name entered: ", fullName);
+
+    if (fullName && fullName.length > 0) {
```

Comment:
nice holistic thinking, accounting for all possible cases, really like this!!! :-)